### PR TITLE
Connectivity adapter: renewal and revocation added

### PR DESCRIPTION
**Pull request by akgalwas**
**Description**

Changes proposed in this pull request:

- Certificates revocation and renewal implemented
- Getting application name and event base url from Director implemented

### CODE REVIEW

**Comment by akgalwas** [comment]
Notes:
- implementation doesn't contain calling Director in signingRequestInfo handle as it requires Connectivity Adapter to be registered as integration system
- Connector Service tests were used to initially check if contract is not broken, however, they require modification to be used as acceptance test (there is a need to create application is Director prior to generating/renewing certificates)

File: components/connectivity-adapter/cmd/main.go [hunk 1]
```diff
@@ -26,23 +26,31 @@ func main() {
...
+	err := connector.RegisterHandler(connectorRouter, cfg.Connector, cfg.AppRegistry.DirectorEndpoint)
 	if err != nil {
-		exitOnError(err, "Failed to init Connector handler")
+		return nil, err
 	}
 
-	log.Printf("Listening on %s", cfg.Address)
-	err = http.ListenAndServe(cfg.Address, router)
-	exitOnError(err, fmt.Sprintf("while listening on %s", cfg.Address))
+	return connectorRouter, nil
```
Comment (aszecowka) [inline components/connectivity-adapter/cmd/main.go]: then `/v1/health` is not accessible anymore

File: components/connectivity-adapter/cmd/main.go [hunk 2]
```diff
@@ -7,14 +7,14 @@ import (
...
-	connector "github.com/kyma-incubator/compass/components/connectivity-adapter/internal/connector"
+	connector "github.com/kyma-incubator/compass/components/connectivity-adapter/internal/connectorservice"
 	"github.com/kyma-incubator/compass/components/connectivity-adapter/pkg/health"
 	"github.com/pkg/errors"
 	"github.com/vrischmann/envconfig"
 )
 
 type config struct {
-	Address string `envconfig:"default=127.0.0.1:8080"`
+	APIAddress string `envconfig:"default=127.0.0.1:8080"`
```
Comment (pkosiec) [inline components/connectivity-adapter/cmd/main.go]: Do we need to rename it? Then we would need to update docs as well and helm charts values

Comment (akgalwas) [reply components/connectivity-adapter/cmd/main.go]: Done

File: components/connectivity-adapter/cmd/main.go (hunk 1 above)
Comment (akgalwas) [reply components/connectivity-adapter/cmd/main.go]: Done

**Pull request merged**